engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/views/content-host-info.html

```diff
@@ -329,23 +312,23 @@ <h4 translate>Networking</h4>
 
       <div class="detail">
         <span class="info-label" translate>Hostname</span>
-        <span class="info-value">{{ contentHostFacts.network.hostname }}</span>
+        <span class="info-value">{{ host.name }}</span>
       </div>
 
       <div class="detail">
         <span class="info-label" translate>IPv4 Address</span>
-        <span class="info-value">{{ contentHostFacts.network.ipv4_address }}</span>
+        <span class="info-value">{{ host.facts["net::interface::eth0::ipv4_address"] }}</span>
       </div>
 
       <div class="detail">
         <span class="info-label" translate>IPv6 Address</span>
-        <span class="info-value">{{ contentHostFacts.network.ipv6_address }}</span>
+        <span class="info-value">{{ host.facts["net::interface::eth0::ipv6_address"] }}</span>
```

Comment:
This isn't really the proper way to do this as eth0 may not a valid interface (for me it wasn't).  'network::ipv4_address' is a fact as well as network::ipv6_address   I would just use those